.github/workflows/issue-label.yml

```
@@ -0,0 +1,57 @@
+name: New issues labeler
+on:
+  issues:
+    types:
+      - opened
+
+permissions:
+  issues: write
+
+jobs:
+  new-contributor-labeler:
+    name: Label issues when they are created by a first time contributor
+    permissions:
+      contents: read
+      issues: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7
```

Could you please use directly the v7?
```suggestion
      - uses: actions/github-script@v7
```